console/account/concepts.mdx

```diff
@@ -14,6 +14,8 @@ categories:
 
 [Abuse](https://www.scaleway.com/en/abuse-notice/) of Scaleway services includes cyber-crime, copyright violation, illegal or offensive content, spamming and malware distribution. Abuse should be [reported in the console](https://console.scaleway.com/support/abuses/create).
 
+Abuses on the same offending resource for the same reason within a given timeframe are grouped by case. An abuse either creates a case if no case is currently open or joins an existing case. Scaleway clients are responsible for taking the appropriate security measures to solve the case and related abuses.
```

Comment:
```suggestion
Multiple instances of abuse from the same offending resource for the same reason within a given timeframe are grouped by case. An abuse report either creates a case if no case is currently open or joins an existing case. Scaleway clients are responsible for taking the appropriate security measures to solve the case and related abuse reports.
```

Reply:
This is tricky because I know "abuse" is often treated as a countable noun at SCW, but generally it's an uncountable noun so it sounds weird to me saying "abuses" / "an abuse". I think AWS also treats it as uncountable: https://aws.amazon.com/blogs/mt/automating-processes-for-handling-and-remediating-aws-abuse-alerts/

I'm not sure when to say "abuse ticket" vs "abuse report" (The reporter creates a report, the abuser receives a ticket?)